EU/main.yaml

```
@@ -0,0 +1,72 @@
+sources:
+  scream:
+    args:
+      chunks: null
+      consolidated: true
+      urlpath: /work/bb1153/k202134/scream-rearranged/scream{{
+        time }}_hp{{ zoom }}_v6.zarr
+    driver: zarr
+    parameters:
+      time:
+        allowed:
+        - 2D_hrly
+        - 3D_ne120
```

We should unify the type of this parameter across models (and very likely should use ISO duration specifiers to encode temporal resolution).
The `time` parameter should also **only** select time (it may be that some variables are missing at certain resolutions though).
```suggestion
        - PT1H
        - PT2H # (?)
```